packages/terra-functional-testing/src/commands/utils/downloadScreenshots.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios').default;
+const extract = require('extract-zip');
+const fs = require('fs-extra');
+const path = require('path');
+const { Logger } = require('@cerner/terra-cli');
+
+const logger = new Logger({ prefix: '[terra-functional-testing:downloadScreenshots]' });
+
+/**
+ * Downloads and unzips the screenshots from provided url.
+ * @param {string} screenshotUrl - The url to the repository that stores the screenshots.
+ */
+async function downloadScreenshots(screenshotUrl) {
+  if (!screenshotUrl) {
+    logger.warn('A url to the screenshot registry is required to download screenshots.');
```

Comment:
should we do an error here instead?
```suggestion
    logger.error('A url to the screenshot registry is required to download screenshots.');
```